src/main/java/org/venice/beachfront/bfapi/model/oauth/GeoAxisCommonName.java

```diff
@@ -0,0 +1,67 @@
+package org.venice.beachfront.bfapi.model.oauth;
+
+import java.io.IOException;
+import java.util.Collections;
+import java.util.LinkedList;
+import java.util.List;
+
+import com.fasterxml.jackson.core.JsonParseException;
+import com.fasterxml.jackson.core.JsonParser;
+import com.fasterxml.jackson.core.JsonProcessingException;
+import com.fasterxml.jackson.core.TreeNode;
+import com.fasterxml.jackson.databind.DeserializationContext;
+import com.fasterxml.jackson.databind.JsonDeserializer;
+import com.fasterxml.jackson.databind.JsonNode;
+
+public interface GeoAxisCommonName {
```

Comment:
Please add some JavaDoc comments here to explain why this class is needed.   Someone looking at this in one year might be super confused as to why this class exists :p

Reply:
Will do.